### ISSUE TITLE
`rbioacc`: an `R`-package to analyze toxicokinetic data

### ISSUE THREAD

**Issue by sandrinecharles** [ropensci/software-review#499]
**Reviewers:** 
Submitting Author Name: Sandrine Charles
Submitting Author Github Handle: @sandrinecharles
Other Package Authors Github handles: (comma separated, delete if none) @aursiber, @virgile-baudrot
Repository:  https://github.com/aursiber/rbioacc
Version submitted:
Submission type: Standard
Editor:  TBD 
Reviewers:  TBD 

Archive: TBD
Version accepted: TBD
Language: en
---

-   Paste the full DESCRIPTION file inside a code block below:

```
Package: rbioacc
Title: Inference and Prediction of ToxicoKinetic (TK) Models
Version: 1.1-0
Authors@R: c(
    person("Virgile", "Baudrot", role = "aut", email = "virgile.baudrot@posteo.net"),
    person("Sandrine", "Charles", role = "aut"),
    person("Ophélia", "Gestin", role = "ctb"),
    person("Mélina", "Kaag", role = "aut"),
    person("Christelle", "Lopes", role = "ctb"),
    person("Gauthier", "Multari", role = "ctb"),
    person("Alain", "Pavé", role = "ctb"),
    person("Aude", "Ratier", role = "aut"),
    person("Aurélie", "Siberchicot", role = c("aut", "cre"), email = "aurelie.siberchicot@univ-lyon1.fr")
    )
Description: The MOSAICbioacc application is a turnkey package providing bioaccumulation
    factors (BCF/BMF/BSAF) from a toxicokinetic (TK) model fitted to
    accumulation-depuration data. It is designed to fulfil the requirements
    of regulators when examining applications for market authorization of active
    substances. See Ratier et al. (2021) <doi:10.1101/2021.09.08.459421>.
URL: https://github.com/aursiber/rbioacc
BugReports: https://github.com/aursiber/rbioacc/issues
License: MIT + file LICENSE
Encoding: UTF-8
LazyData: true
Roxygen: list(markdown = TRUE)
RoxygenNote: 7.1.2
Biarch: true
Depends: 
    R (>= 3.5.0)
Imports: 
    ggplot2,
    methods,
    Rcpp,
    rstan (>= 2.18.1),
    rstantools (>= 2.1.1),
    ggmcmc,
    GGally,
    loo,
    stringr,
    stats,
    zoo
LinkingTo: 
    BH (>= 1.66.0),
    Rcpp,
    RcppEigen (>= 0.3.3.3.0),
    RcppParallel (>= 5.0.1),
    rstan (>= 2.18.1),
    StanHeaders (>= 2.18.0)
SystemRequirements: GNU make
Config/testthat/edition: 3
Suggests: 
    knitr,
    rmarkdown,
    testthat
VignetteBuilder: knitr

```

## Scope

- Please indicate which category or categories from our [package fit policies](https://ropensci.github.io/dev_guide/policies.html#package-categories) this package falls under: (Please check an appropriate box below. If you are unsure, we suggest you make a pre-submission inquiry.):

	- [ ] data retrieval
	- [ ] data extraction
	- [ ] data munging
	- [ ] data deposition
	- [X] workflow automation
	- [ ] version control
	- [ ] citation management and bibliometrics
	- [X] scientific software wrappers
	- [X] field and lab reproducibility tools
	- [ ] database software bindings
	- [ ] geospatial data
	- [ ] text analysis

- Explain how and why the package falls under these categories (briefly, 1-2 sentences):
The `rbioacc` package provides ready-to-use functions to fit and predict accumulation and depuration of chemical substances within organisms, based on one-compartment toxicokinetics models. This package supports the freely available on-line web platform MOSAIC_bioacc (https://mosaic.univ-lyon1.fr/bioacc).

-   Who is the target audience and what are scientific applications of this package? 
The target audience is academic research, regulatory bodies and chemical companies. Indeed, this package provide all necessary tools to estimate bioaccumulation capacity of a any chemical substance when a given living species is exposed to.

-   Are there other R packages that accomplish the same thing? If so, how does yours differ or meet [our criteria for best-in-category](https://ropensci.github.io/dev_guide/policies.html#overlap)?
To our knowledge, no other package with the same feature exists.

-   (If applicable) Does your package comply with our [guidance around _Ethics, Data Privacy and Human Subjects Research_](https://devguide.ropensci.org/policies.html#ethics-data-privacy-and-human-subjects-research)?
Yes

-   If you made a pre-submission inquiry, please paste the link to the corresponding issue, forum post, or other discussion, or @tag the editor you contacted.

-   Explain reasons for any [`pkgcheck` items](https://docs.ropensci.org/pkgcheck/) which your package is unable to pass.

## Technical checks

Confirm each of the following by checking the box.

- [X] I have read the [guide for authors](https://devguide.ropensci.org/guide-for-authors.html) and [rOpenSci packaging guide](https://devguide.ropensci.org/building.html).

This package:

- [X] does not violate the Terms of Service of any service it interacts with.
- [X] has a CRAN and OSI accepted license.
- [X] contains a [README with instructions for installing the development version](https://ropensci.github.io/dev_guide/building.html#readme).
- [X] includes [documentation with examples for all functions, created with roxygen2](https://ropensci.github.io/dev_guide/building.html#documentation).
- [X] contains a vignette with examples of its essential functions and uses.
- [X] has a [test suite](https://ropensci.github.io/dev_guide/building.html#testing).
- [ ] has [continuous integration](https://ropensci.github.io/dev_guide/ci.html), including reporting of test coverage using services such as Travis CI, Coveralls and/or CodeCov.

## Publication options

- [X] Do you intend for this package to go on CRAN?
- [ ] Do you intend for this package to go on Bioconductor?

- [X] Do you wish to submit an Applications Article about your package to [Methods in Ecology and Evolution](http://besjournals.onlinelibrary.wiley.com/hub/journal/10.1111/(ISSN)2041-210X/)? If so:

<details>
<summary>MEE Options</summary>

- [X] The package is novel and will be of interest to the broad readership of the journal.
- [ ] The manuscript describing the package is no longer than 3000 words.
- [X] You intend to archive the code for the package in a long-term repository which meets the requirements of the journal (see [MEE's Policy on Publishing Code](http://besjournals.onlinelibrary.wiley.com/hub/journal/10.1111/(ISSN)2041-210X/journal-resources/policy-on-publishing-code.html))
- (*Scope: Do consider MEE's [Aims and Scope](http://besjournals.onlinelibrary.wiley.com/hub/journal/10.1111/(ISSN)2041-210X/aims-and-scope/read-full-aims-and-scope.html) for your manuscript. We make no guarantee that your manuscript will be within MEE scope.*)
- (*Although not required, we strongly recommend having a full manuscript prepared when you submit here.*)
- (*Please do not submit your package separately to Methods in Ecology and Evolution*)

</details>

## Code of conduct

- [X] I agree to abide by [rOpenSci's Code of Conduct](https://ropensci.org/code-of-conduct/) during the review process and in maintaining my package should it be accepted.

**Comment by mpadge** [comment]
@sandrinecharles Sorry about that, we are trying to diagnose what went wrong. Checks should appear soon.

**Comment by sandrinecharles** [comment]
@mpadge no problem, this is not really urgent. By the way, I already submitted to MEE, and only after I saw that it is not recommended to do in this order. Sorry...

**Comment by emilyriederer** [comment]
Hi there, @sandrinecharles ! I wanted to follow up on this issue. Were you still interested in addressing some of @ldecicco-USGS 's feedback and moving towards the review process? I added the "holding" tag since there hasn't been much recent discussion on this issue, but happy to continue the conversation as you wish. Thanks!

**Comment by sandrinecharles** [comment]
Hi @emilyriederer! Yes, we still want to consider feedback, but the engineer that should be in charge of this has quick how lab. This work is thus temporally suspended.

**Comment by emilyriederer** [comment]
Thanks for the update, @sandrinecharles ! We will leave this on hold for now.

**Comment by emilyriederer** [comment]
Hi @sandrinecharles  ! We are doing a sweep of stale review issues. Since this review has been open and inactive for so long, much may have changed including author, editor, and reviewer bandwidth and ever-evolving rOpenSci best practices. As such, I'm closing this issue. If you still have interest and capacity, we would welcome you to open a new submission issue!
